applications/ShapeOptimizationApplication/test_examples/01_Strain_Energy_Minimization_3D_Hook/optimization_parameters.json

```diff
@@ -83,28 +85,27 @@
             }
         },
         "optimization_algorithm" : {
-            "name"                                 : "penalized_projection",
-            "correction_scaling"                   : 0.1,
-            "use_adaptive_correction"              : true,
-            "max_iterations"                       : 50,
-            "relative_tolerance"                   : 1e-0,
-            "project_gradients_on_surface_normals" : true
-        },
-        "line_search" : {
-            "line_search_type"           : "manual_stepping",
-            "normalize_search_direction" : true,
-            "step_size"                  : 3
+            "name"                    : "penalized_projection",
+            "correction_scaling"      : 0.1,
+            "use_adaptive_correction" : true,
+            "max_iterations"          : 50,
+            "relative_tolerance"      : 1e-0,
+            "line_search" : {
+                "line_search_type"           : "manual_stepping",
+                "normalize_search_direction" : true,
+                "step_size"                  : 3.0
+            }
         },
         "output" : {
             "output_directory"        : "Optimization_Results",
             "response_log_filename"   : "response_log",
             "design_history_filename" : "design_history",
             "design_output_mode"      : "WriteOptimizationModelPart",
             "nodal_results" : [ "NORMALIZED_SURFACE_NORMAL",
-                                "OBJECTIVE_SENSITIVITY",
-                                "CONSTRAINT_SENSITIVITY",
-                                "MAPPED_OBJECTIVE_SENSITIVITY",
-                                "MAPPED_CONSTRAINT_SENSITIVITY",
+                                "DF1DX",
```

Comment:
it would be nice to have an automated way to output all objective and constraint sensitivities, simply with a flag.

Reply:
Yep. We should consider this in future developments.